include/GafferImage/CDL.h

```diff
@@ -55,6 +55,14 @@ class GAFFERIMAGE_API CDL : public OpenColorIOTransform
 
 		GAFFER_NODE_DECLARE_TYPE( GafferImage::CDL, CDLTypeId, OpenColorIOTransform );
 
+		// Required due to the change between the TRANSFORM_DIR_FORWARD and
+		// TRANSFORM_DIR_INVERSE enum values between OCIO 1.0 and 2.0
+		enum Direction
+		{
+			Forward = 1,
+			Inverse = 2
+		};
```

Comment:
I'm confused because you seem to say that option 2 wouldnt be that bad and only very limited case would create a backward compatibility issue and it would be cleaner ( single enum ) moving forward. 

But then, you say you prefer option 1 with multiple enums?

I will talk to pipeline and lighting at IE that might be the one using CDL the most and come back to you but my feeling is the backward compatibility problem is such an edge case and the consequences would be pretty obvious and easy to fix that option 2 would be better.

Reply:
Sounds good to me, happy to update those enum values to be consolidated at a later date.

Reply:
Those have now been changed to match LUT and LookTransform.